phases/phase-1/challenges/refer.md

```diff
@@ -0,0 +1,38 @@
+# Refer a friend
+:::tip Note  
+This challenges allows you to refer one or more friend(s) and get **200 Desmos Tokens per invited user that has completed the [Post challenge](create-post.md) and [accepted your referral](accept-referral.md)**.  
+
+Please note that you can refer multiple friends and get 200 Desmos Tokens per friend referred.
+
+If one of the users who have referred does **not** create a post or accept your referral, you will **not** be granted any Desmos Token. 
+:::
+
+Referring a friend is the way that you have to let people know about Desmos, helping us increase the user base. In order to complete this challenge follow this procedure: 
```

Comment:
Referring a friend is the way that you have to let people know about Desmos, helping us increase the user base. In order to complete this challenge follow this procedure: 

->

Referral program is a great way for us to go viral with more co-builders like you! To create a censorship-resistant social network, we need your help to let more people know about Desmos and increase its user base.  In order to complete this challenge, please follow this procedure: